src/sentry/api/endpoints/sentry_app_components.py

```diff
@@ -27,10 +27,12 @@ def get(self, request, sentry_app):
 class OrganizationSentryAppComponentsEndpoint(OrganizationEndpoint):
     @add_integration_platform_metric_tag
     def get(self, request, organization):
+        project_id = request.GET["projectId"]
```

Comment:
@mgaeta would it make more sense to do `request.GET.get("projectId")` in case the key is missing?